configmaptocrs/main.go

```diff
@@ -76,7 +83,7 @@ func generate(w io.Writer, origin string) error {
 		return err
 	}
 
-	log.Println("Validating the custom resources")
+	log.Println("Checking the resources are parsed correctly")
```

Comment:
does this / other log.Println show up in stdout when using -stdout?

Reply:
Thanks for the review! Yes, it shows. And I made sure the logs don't mix up with the result YAML.
Plus I'll note that I made this change to the log text because the
following line is: _, err = config.For(resources, config.**DontValidate**)
and the new log text will prevent confusion.
Here is an example of the output on stdout:
```
2022/08/30 06:44:51 Reading configmap
2022/08/30 06:44:51 Decoding configmap
2022/08/30 06:44:51 Creating custom resources
2022/08/30 06:44:51 Checking the resources are parsed correctly
2022/08/30 06:44:51 Creating the output YAML
# This was autogenerated by MetalLB's custom resource generator.
apiVersion: metallb.io/v1beta1
kind: IPAddressPool
metadata:
  creationTimestamp: null
  name: default
  namespace: metallb-system
spec:
  addresses:
  - 192.168.1.240-192.168.1.250
status: {}
---
apiVersion: metallb.io/v1beta1
kind: L2Advertisement
metadata:
  creationTimestamp: null
  name: l2advertisement1
  namespace: metallb-system
spec:
  ipAddressPools:
  - default
status: {}
---
pod "configmaptocrs" deleted
```